packages/blocks/README.md

```diff
@@ -16,169 +18,6 @@ npm install @wordpress/blocks --save
 
 _This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for such language features and APIs, you should include [the polyfill shipped in `@wordpress/babel-preset-default`](https://github.com/WordPress/gutenberg/tree/HEAD/packages/babel-preset-default#polyfill) in your code._
 
-## Getting Started
```

Comment:
This is actually quite a nice tutorial. I appreciate that this is probably not the right place for it as it's quite hard to find as it's buried deep in the docs. It would be nice to retain it in some form though. Either move it to the Getting Started section as an alternative tutorial, or use this content as the basis of a 'Write your first block' post on the Developer Blog.

Reply:
This tutorial is outdated. It was authored in 2017, and everything has changed since then. We can bring it back in a different place as a follow-up after applying all necessary edits so it plays it’s purpose correctly.

Reply:
Agreed with both; it would be nice to have it up to date and in a better location, but that's outside the scope of this PR.

Reply:
> use this content as the basis of a 'Write your first block' post on the Developer Blog.

That sounds like a good idea.

> I also think that the example project here is nicer and more functional than the [Getting Started one](https://developer.wordpress.org/block-editor/getting-started/create-block/), which is a bit rudimentary and the font used only works in Firefox.

In some ways, it's better, but still, it isn't good enough for what we want to promote these days:
- block supports usage for global styles is not covered, it would probably be better that all the hassle with CSS files
- we should start promoting dynamic blocks, or even better hybrid blocks, where we use `save` as the fallback for the block, but we still enrich the content with PHP
- in a few months, the tutorial should also cover how to make blocks interactive on the front end

That's a very interesting topic that we should discuss as the block editor handbook evolves. There is value in keeping Create Block Tutorial simple, so it won't be simple to find the sweet spot. The interesting part is that some of the How-To Guides could fill the full picture of block development and extend the block crafted in the tutorial with additional functionalities.